packages/et-mfr/src/Adaptor.js

```diff
@@ -0,0 +1,149 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils.js';
+
+/**
+ * State object
+ * @typedef {Object} HttpState
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * Options provided to the HTTP request
+ * @typedef {Object} RequestOptions
+ * @public
+ * @property {object|string} body - body data to append to the request.
+ * @property {object} query - An object of query parameters to be encoded into the URL.
+ * @property {object} headers - An object of headers to append to the request.
+ * @property {string} parseAs - Parse the response body as json, text or stream. By default will use the response headers.
+ */
+
+/**
+ * Make a GET request to Ethiopia MFR
+ * @example <caption>Get all regions</caption>
+ * get('/Location/Regions');
+ * @example <caption>Get a facility with id</caption>
+ * get('Facility', {
+ *   query: {
+ *     id: '1071644',
+ *   },
+ * });
+ * @example <caption>Get all facility details with search options</caption>
+ * get('Facility/All', {
+ *   query: {
+ *     name: '010 Kombolcha Health Post',
+ *   },
+ * });
+ * @function
+ * @public
+ * @param {string} path - Path to resource
+ * @param {RequestOptions} options - Optional request options
+ * @returns {Operation}
+ * @state {HttpState}
+ */
+export function get(path, options) {
+  return request('GET', path, null, options);
+}
+
+/**
+ * Make a POST request to Ethiopia MFR
+ * @example <caption>Retrieve facilities with pagination</caption>
+ * post('Facility/GetFacilities', {
+ *   pageNumber: 1,
+ *   showPerPage: 25,
+ * });
+ * @example <caption>Retrieve facility details in CSV format</caption>
```

Comment:
Well we  are returning text, Maybe  we should say `Retrieve facility details as CSV strings`